src/cmd/kubectl.rs

```diff
@@ -1036,14 +1037,14 @@ pub fn kubectl_get_crash_looping_pods<P>(
 where
     P: AsRef<Path>,
 {
-    let restarted_min = restarted_min_count.unwrap_or_else(|| 10usize);
+    let restarted_min = restarted_min_count.unwrap_or(10usize);
     let pods = kubectl_exec_get_pods(kubernetes_config, namespace, selector, envs)?;
 
     Ok(pods
         .items
         .into_iter()
         .filter(|pod| {
-            pod.status.phase != KubernetesPodStatusPhase::Running
+            pod.status.reason == KubernetesPodStatusReason::BackOff
                 && pod.status.container_statuses.as_ref().is_some()
```

Comment:
is it useful to check the status then? I'm not sure, if it's backofff, then...

Reply:
It is in case we want to kill if N restarts

Reply:
May be you can add a comment, because it's not obvious

Reply:
Added a comment, hpefuly it's clearer, let me know.